src/DynamoCore/Search/NodeSearchModel.cs

```diff
@@ -14,6 +15,10 @@ namespace Dynamo.Search
     /// </summary>
     public class NodeSearchModel : SearchLibrary<NodeSearchElement, NodeModel>
     {
+        internal NodeSearchModel(ILogger logger = null) : base(logger)
+        {
```

Comment:
even though these are internal - what do you think about adding some comments on why these constructors were added?